docs/tutorial/security.md

```diff
@@ -0,0 +1,34 @@
+# Security, Native Capabilities, and Your Responsibility
+
+As web developers, we usually enjoy the strong security net of the browser - the risks associated with the code we write is relatively small. We rely on the fairly limited amount of power and capabilities granted to a website – and trust that our users enjoy a browser built by a large team of engineers that is able to quickly respond to newly discovered security threats.
+
+When working with Electron, it is important to understand that Electron is not a web browser. It allows you to build powerful desktop apps with web technologies. Its core feature is the ability to build software is just as powerful as completely native applications, eclipsing the limited feature set of a website. The inherent risks scale with the additional powers granted to your code.
+
+With that in mind, be aware that displaying arbitrary content from untrusted sources poses a severe security risk that Electron is not built to handle. In fact, the most popular Electron apps (Atom, Slack, Visual Studio Code, etc) display local content – if your application executes code from an online source, it is your responsibility to ensure that the code is not malicious.
+
+## Chromium Security Issues and Upgrades
+
+While Electron strives to support new versions of Chromium as soon as possible, developers should be aware that upgrading is a serious undertaking - involving hand-editing dozens or even hundreds of files. Given the resources and contributions available today, Electron will often not be on the very latest version of Chromium, lagging behind by either days or weeks.
+
+We feel that our current system of updating the Chromium component strikes an appropriate balance between the resources we have available and the needs of the majority of applications built on top of the framework. We definitely are interested in hearing more about specific use cases from the people that build things on top of Electron. Pull requests and contributions supporting this effort are always very welcome.
+
+## Ignoring Above Advice
+A security issue exists whenever you receive code from a remote destination and execute it locally. As an example, consider a remote website being displayed inside a browser window. If an attacker somehow manages to change said content (either by attacking the source directly, or by sitting between your app and the actual destination), they will be able to execute native code on the user's machine.
+
+> :warning: Under no circumstances should you put some Node code into a JavaScript file, put it on a server somewhere, and then run it from the app. It's a horrible idea. It's a security nightmare.
+
+This is not bulletproof, but at the least, you should attempt the following:
```

Comment:
This would be a great place to explain using `webview` to sandbox remote code and to advise people to only run installed (local) code with node integration enabled